docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc

```diff
@@ -125,16 +139,14 @@ quarkus.http.auth.permission.deny1.paths=/public/*
 quarkus.http.auth.permission.deny1.policy=deny
 ----
 
-TIP: Given the above permission set, `GET /public/foo` would match both permission sets' paths,
-but because it matches the `permit1` permission set's explicit method, `permit1` will be chosen and the request will
-be accepted. `PUT /public/foo` on the other hand, will not match the method permissions of `permit1` and so
-`deny1` will be activated and reject the request.
+TIP: Given the above permission set, `GET /public/foo` would match both permission sets' paths, but because it matches the `permit1` permission set's explicit method, `permit1` will be chosen, and the request will
+be accepted.
+`PUT /public/foo`, on the other hand, will not match the method permissions of `permit1`, so `deny1` will be activated and reject the request.
 
 === Matching multiple paths and methods: both win
 
-If multiple permission sets specify the same path and method (or multiple have no method) then both permissions have to
-allow access for the request to proceed. Note that for this to happen both have to either have specified the method, or
-have no method, method specific matches take precedence as stated above:
+If multiple permission sets specify the same path and method, or multiple has no method, then both permissions have to allow access for the request to proceed.
```

Comment:
Not really sure what the "or multiple has no method" is trying to say, which makes understanding the sentence kinda tricky.

Reply:
Updating:
When multiple permission sets win at the same time per the previously described rules, all these permissions have to allow access for the request to proceed.